diff -Nru lha-114f/Makefile lha-114f.new/Makefile
--- lha-114f/Makefile	Fri Mar  3 20:03:06 2000
+++ lha-114f.new/Makefile	Fri Mar  3 20:03:29 2000
@@ -20,14 +20,15 @@
 
 #MACHINE     = -DSYSTIME_HAS_NO_TM -DFTIME -DEUC
 #MACHINE     = -DSYSTIME_HAS_NO_TM -DEUC -DHAVE_TM_ZONE -DSYSV_SYSTEM_DIR
-MACHINE     = -DSYSTIME_HAS_NO_TM -DEUC -DSYSV_SYSTEM_DIR -DMKTIME
+MACHINE  = -DEUC -DSYSV_SYSTEM_DIR -DTZSET -DARCHIVENAME_EXTENTION=\".lha\" \
+	 -DBACKUPNAME_EXTENTION=\".bak\" -DSUPPORT_LH6
 
 #OPTIMIZE	= -O2 -fstrength-reduce -fomit-frame-pointer
 OPTIMIZE	= -O2 -DSUPPORT_LH6
 
-BINDIR = /usr/local/bin
-MANDIR = /usr/local/man
-MANSECT = n
+BINDIR = /usr/bin
+MANDIR = /usr/man
+MANSECT = 1
 
 INSTALL = install
 INSTALLBIN = -s -m 755
